features/cilium/cilium-01.feature

```diff
@@ -0,0 +1,90 @@
+# doc1:   https://gitlab.suse.de/mkravec/scripts/blob/master/tests/cilium.sh
+# doc2:   http://docs.cilium.io/en/v1.6/gettingstarted/http/
+# doc3:   https://github.com/fgerling/bdd-poc
+
+Feature: cilium-basic
+
+ Scenario: Test-Cilium-Basic on Skuba Cluster
+    Given "skuba" exist in gopath
+    #And VARIABLE "work-folder" equals "/Users/alexeitighineanu/go/src/github.com/fgerling/bdd-poc/imba-cluster"
+    When I run "skuba cluster status"       
+    Then the output contains "master" and "worker"
+    When I run "kubectl get all --namespace=kube-system"
+    Then the output contains "cilium" and "dex"
+
+    Scenario: Deploy the starwars cilium pods
+    When I run "kubectl create -f https://raw.githubusercontent.com/cilium/cilium/v1.6/examples/minikube/http-sw-app.yaml"
+    And wait "10 seconds"
+    When I run "kubectl get pods --selector=org=empire" 
+    Then the output contains "deathstar" and "tiefighter"
+    When I run "kubectl get pods --selector=org=alliance"
+
```

Comment:
I think one `Then` is missing here

Reply:
You are right @lcavajani. @atighineanu could you have a look at it?

Reply:
corrected